modules/osd-aws-privatelink-required-resources.adoc

```diff
@@ -1,10 +1,21 @@
 // Module included in the following assemblies:
 //
 // * rosa_install_access_delete_clusters/rosa-aws-privatelink-creating-cluster.adoc
-[id="osd-aws-privatelink-required-resources.adoc_{context}"]
+// * rosa_hcp/rosa-hcp-aws-privatelink-creating-cluster.adoc
+ifeval::["{context}" == "rosa-hcp-aws-privatelink-creating-cluster"]
+:rosa-hcp:
+endif::[]
+ifeval::["{context}" == "rosa-aws-privatelink-creating-cluster"]
+:rosa-standalone:
+endif::[]
+[id="osd-aws-privatelink-required-resources_{context}"]
 = Requirements for using AWS PrivateLink clusters
-
+ifdef::rosa-hcp[]
+For {hcp-title} private clusters, internet gateways, NAT gateways and public subnets are not required, but the private subnets must have internet connectivity provided to install required components. At least one single private subnet is required. The following table shows the AWS resources that are required for a successful installation:
```

Comment:
Is "single private subnet" something specific, or could this be "one" instead of "one single"?